pom.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I needed somewhere to stand this up when I published a SNAPSHOT release to our organization's internal Nexus repository. You may have a better way to do this, but I needed to turn this around in a hurry.